inc/saiswitch.h

```diff
@@ -2141,6 +2141,14 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP,
 
+    /**
+     * @brief Up to 4 TPID can be configured on this switch to any port/LAG
+     *
+     * @type bool
+     * @flags READ_ONLY
```

Comment:
who enforce the limit of 4 ? SAI or Application

Reply:
This switch attribute "SAI_SWITCH_ATTR_TPID_CONFIGURABLE" is the way for SAI vendor to tell the application that the SAI vendor is able to handle 4 or more TPID configuration for this switch.

Reply:
Once SAI indicate it is able to handle TPID config, the application will ensure that it will not exceed more than 4 TPID config (default 0x8100 count towards this count) for the given switch.  Application will reject any TPID config attempt if this feature is not supported by SAI for the given switch.